src/Containers/List/ListHeader.tsx

```diff
@@ -82,21 +82,32 @@ interface IResponsiveSearchProps {
     isExpanded: boolean;
 }
 
-const SearchContainer = styled.div<IResponsiveSearchProps>`
+const SearchContainer = styled.div`
     ${Mixins.flex('row')}
     margin-left: auto;
 `;
 
 const IconContainer = styled.div<IResponsiveSearchProps>`
     ${Mixins.flex('center')}
-    margin-left: 5px;
+    
 `;
 const HeadingContainer = styled.div<IResponsiveSearchProps>`
     ${Mixins.transition(['transform', 'opacity'])}
+    @keyframes heading {
+        0% {
+            position: fixed;
+        }
+        90% {
+            position: fixed;
+        }
+        100% {
+            position: relative;
```

Comment:
Why is there position fixed in the first place?

Reply:
To remove it from the stacking context?

## Without Fixed
![Out of stacking](https://user-images.githubusercontent.com/29719870/114280941-73447880-99f0-11eb-9def-54bd53036452.gif)